cmd/plugin/cli/uninstall.go

```diff
@@ -167,6 +176,8 @@ func setUninstallValues(cmd *cobra.Command, config *apiv1.KubectlStorageOSConfig
 	config.Spec.Uninstall.EtcdOperatorYaml = viper.GetString(installer.UninstallEtcdOperatorYamlConfig)
 	config.Spec.Uninstall.EtcdClusterYaml = viper.GetString(installer.UninstallEtcdClusterYamlConfig)
 	config.Spec.Uninstall.ResourceQuotaYaml = viper.GetString(installer.UninstallResourceQuotaYamlConfig)
+	config.Spec.IncludeLocalPathProvisioner = viper.GetBool(installer.IncludeLocalPathProvisionerFlag)
+	config.Spec.Uninstall.LocalPathProvisionerYaml = viper.GetString(installer.LocalPathProvisionerYamlFlag)
```

Comment:
Same as above, we should fetch `...Config` values at this point

Reply:
"above" refers to https://github.com/storageos/kubectl-storageos/pull/163#discussion_r848327474 which got lost in my review